### PR TITLE
Update sensitivity-labels-sharepoint-onedrive-files.md

### PR DESCRIPTION
**Pull request by msbemba**
Added step to update the Module if downloaded from PS gallery

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md [hunk 1]
```diff
@@ -72,17 +72,21 @@ For this preview, use the OneDrive sync app version 19.002.0121.0008 or later on
 
 Before you enable the preview, ensure that you're running the latest SharePoint Online Management Shell. If you already have the latest version, you can go ahead and enable the preview.
 
-To prepare the SharePoint Online Management Shell for the preview:
+1. If you installed a previous version of the SharePoint Online Management Shell from Powershell Gallery, You can Update the module by running the below cmdlet
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md]:
```suggestion
1. If you have installed a previous version of the SharePoint Online Management Shell from PowerShell gallery, you can update the module by running the following cmdlet.
```

File: microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md [hunk 2]
```diff
@@ -72,17 +72,21 @@ For this preview, use the OneDrive sync app version 19.002.0121.0008 or later on
...
 
-1. If you installed a previous version of the SharePoint Online Management Shell, go to **Add or remove programs** and uninstall “SharePoint Online Management Shell.”
+    ```PowerShell
+    Update-Module -Name Microsoft.Online.SharePoint.PowerShell
+    ```
+
+2. Else, If you installed a previous version of the SharePoint Online Management Shell from the Microsoft download center, go to **Add or remove programs** and uninstall “SharePoint Online Management Shell.”
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md]:
```suggestion
2. Alternatively, if you have installed a previous version of the SharePoint Online Management Shell from the Microsoft Download Center, you can also go to **Add or remove programs** and uninstall the SharePoint Online Management Shell.
```

File: microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md [hunk 3]
```diff
@@ -72,17 +72,21 @@ For this preview, use the OneDrive sync app version 19.002.0121.0008 or later on
...
+3. In a web browser, go to the Download Center page and [Download the latest SharePoint Online Management Shell](https://go.microsoft.com/fwlink/p/?LinkId=255251).
 
-3. Select your language and then click **Download**.
+4. Select your language and then click **Download**.
 
-4. Choose between the x64 and x86 .msi file. Download the x64 file if you run the 64-bit version of Windows or the x86 file if you run the 32-bit version. If you don’t know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443/windows-which-operating-system).
+5. Choose between the x64 and x86 .msi file. Download the x64 file if you run the 64-bit version of Windows or the x86 file if you run the 32-bit version. If you don’t know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443/windows-which-operating-system).
 
-5. After you download the file, run the file and follow the steps in the Setup Wizard.
+6. After you download the file, run the file and follow the steps in the Setup Wizard.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md]:
```suggestion
6. After you have downloaded the file, run the file and follow the steps in the Setup Wizard.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

File: microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md [hunk 4]
```diff
@@ -72,17 +72,21 @@ For this preview, use the OneDrive sync app version 19.002.0121.0008 or later on
...
 
-2. In a web browser, go to the Download Center page and [Download the latest SharePoint Online Management Shell](https://go.microsoft.com/fwlink/p/?LinkId=255251).
+3. In a web browser, go to the Download Center page and [Download the latest SharePoint Online Management Shell](https://go.microsoft.com/fwlink/p/?LinkId=255251).
 
-3. Select your language and then click **Download**.
+4. Select your language and then click **Download**.
 
-4. Choose between the x64 and x86 .msi file. Download the x64 file if you run the 64-bit version of Windows or the x86 file if you run the 32-bit version. If you don’t know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443/windows-which-operating-system).
+5. Choose between the x64 and x86 .msi file. Download the x64 file if you run the 64-bit version of Windows or the x86 file if you run the 32-bit version. If you don’t know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443/windows-which-operating-system).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md]:
```suggestion
5. Choose between the x64 and x86 .msi file. Download the x64 file if you run the 64-bit version of Windows or the x86 file if you run the 32-bit version. If you don’t know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443/windows-which-operating-system)

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - one more

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #716

Thank you.

cc: @kenwith

**Comment by cabailey** [comment]
Approving this change

**Pull request merged**